index.html

```diff
@@ -4310,33 +4310,29 @@ <h2>
 
     <section>
         <h2>
-Persistence of DIDs
+DIDs as URNs
         </h2>
 
         <p>
-Although a <a>DID</a> is able to function as a Uniform Resource Name (URN)—a
-"persistent, location-independent resource identifier"as defined by [[RFC8141]]—
-this is not a requirement of all DIDs. Rather the persistence of a <a>DID</a>,
-is determined by the <a>DID controller</a>.
-
-If a <a>DID controller</a> intends for a <a>DID</a> to function as a URN, there
-are two important security considerations:
+If desired by a <a>DID controller</a>, a <a>DID</a> is capable of fulfilling
+the functions of a Uniform Resource Name (URN) as defined by [[RFC8141]], i.e.,
+"a persistent, location-independent resource identifier". A <a>DID controller</a>
+who intends to use a <a>DID</a> for this purpose is advised to follow the
+security considerations in [[RFC8141]]. In particular:
```

Comment:
Since I can't suggest over a suggestion, but I can make a multi-line suggestion --
```suggestion
If desired by a <a>DID controller</a>, a <a>DID</a> is capable of acting
as an enhanced Uniform Resource Name (URN) as defined by [[RFC8141]], i.e.,
"a persistent, location-independent resource identifier". DIDs used in this
way provide a cryptographically secure, location-independent identifier for
a digital resource, while also providing the metadata that allows retrieval.
Because of the indirection between the DID Document and the DID itself, the
DID Controller can adjust the actual location of the resource &mdash; or
even provide the resource directly &mdash; without adjusting the DID. DIDs
of this type can definitively verify that the resource retrieved is, in
fact, the resource identified.
        </p>
        <p>
A <a>DID controller</a> who intends to use a <a>DID</a> for this purpose is
advised to follow the security considerations in [[RFC8141]]. In particular:
```

Reply:
@TallTed I incorporated your suggestions. I realized my last push still had the bullets as a numbered list. I will fix that now.